HOC/StyledMainHeader.tsx

```diff
@@ -26,15 +26,15 @@ const StyledMainHeader: React.FC<HighlightedHeaderProps> = ({
 
   return (
     <h1
-      className={`bg-black-300 text-[1.6rem] font-bold leading-[3.0rem] tracking-[-0.42px] text-white-900 sm:text-[2.625rem] lg:text-[3.8rem] lg:leading-[5.2rem] lg:tracking-[0.64px] ${className}`}
+      className={`bg-black-300 px-[1.5rem] text-center text-[2.625rem] font-bold leading-[115%] tracking-[-0.42px] text-white-900 md:px-[5.4rem] md:text-[4rem]  md:leading-[130%] md:tracking-[0.64px] ${className}`}
```

Comment:
Can you stick with pixels or rem? Especially since you're using percentages as well. You probably just overlooked this.
`tracking-[-0.42px]`